package.json

```diff
@@ -27,6 +27,9 @@
     "@snowcoders/sortier": "2.6.1",
     "babel-jest": "24.7.1",
     "babel-preset-gatsby": "0.1.11",
+    "cross-env": "^5.2.0",
```

Comment:
```suggestion
    "cross-env": "5.2.0",
```

Reply:
my bad...i missed it...i'll correct it as soon as possible @zainfathoni